netci.groovy

```diff
@@ -46,13 +46,13 @@ def addPushJob(String project, String branch, String os, String configuration)
     Utilities.addGithubPushTrigger(newJob);
 }
 
-["RHEL7.2", "Windows_NT", "CentOS7.1"].each { os ->
+["Ubuntu14.04", "Ubuntu16.04", "Ubuntu16.10", "Fedora24", "Debian8.4", "RHEL7.2", "Windows_NT", "CentOS7.1", "OSX10.12"].each { os ->
   addPullRequestJob(project, branch, os, "Release", true);
   addPullRequestJob(project, branch, os, "Debug", false);
 };
 
 // Per push, run all the jobs
-["Ubuntu14.04", "RHEL7.2", "Windows_NT", "CentOS7.1", "OSX10.12"].each { os ->
+["Ubuntu14.04", "Ubuntu16.04", "Ubuntu16.10", "Fedora24", "Debian8.4", "RHEL7.2", "Windows_NT", "CentOS7.1", "OSX10.12"].each { os ->
```

Comment:
Ubuntu16.10 is EOL we should remove it. We should also consider removing Ubuntu14.04

Reply:
Removed.